lib/permutations.rb

```diff
@@ -1,4 +1,31 @@
-
 def permutations?(string1, string2)
-  raise NotImplementedError, "permutations? not implemented" 
+  array_1 = string1.split(//)
+  array_2 = string2.split(//)
+  hash = {}
+
+  if array_1.length == 0 && array_2.length == 0
+    return true
+  elsif array_1.length != array_2.length
+    return false
+  end
+
+  array_1.each do |element|
+    if array_1.count(element) != array_2.count(element)
+      return false
+    end
+  end
```

Comment:
This section here is O(n^2), because `count` is O(n).  Can you think of a better way?